libs/utils/platforms/hikey960.json

```diff
@@ -0,0 +1,30 @@
+{
+    "board" : {
+	"cores" : [
+	    "A53", "A53", "A53", "A53", 
+	    "A73", "A73", "A73", "A73"
+	],
+	"big_core" : "A73",
+	"modules" : ["bl", "cpufreq", "cpuidle"]
+    },
+    "nrg_model": {
```

Comment:
Afaik they should: https://android.googlesource.com/kernel/hikey-linaro/+/android-hikey-linaro-4.4/arch/arm64/boot/dts/hisilicon/hi3660-sched-energy.dtsi